README.md

```diff
@@ -15,68 +11,95 @@ module "bootstrap" {
+| default\_region | Default region to create resources where applicable. | string | n/a | yes |
+| group\_billing\_admins | Google Group for GCP Billing Administrators | string | n/a | yes |
+| group\_org\_admins | Google Group for GCP Organization Administrators | string | n/a | yes |
+| org\_admins\_org\_iam\_permissions | List of permissions granted to the group supplied in group_org_admins variable across the GCP organization. | list(string) | `[ "roles/billing.user", "roles/resourcemanager.organizationAdmin", "roles/resourcemanager.projectCreator" ]` | no |
+| organization\_id | GCP Organization ID | string | n/a | yes |
+| project\_prefix | Name prefix to use for projects created. | string | `"cft"` | no |
+| sa\_enable\_impersonation | Allow org_admins group to impersonate service account & enable APIs required. | bool | `"false"` | no |
+| sa\_org\_iam\_permissions | List of permissions granted to Terraform service account across the GCP organization. | list(string) | `[ "roles/billing.user", "roles/compute.networkAdmin", "roles/compute.xpnAdmin", "roles/iam.serviceAccountAdmin", "roles/logging.configWriter", "roles/orgpolicy.policyAdmin", "roles/resourcemanager.folderCreator", "roles/resourcemanager.folderViewer", "roles/resourcemanager.organizationViewer" ]` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| bucket\_name |  |
+| gcs\_bucket\_tfstate | Bucket used for storing terraform state for foundations pipelines in seed project. |
+| seed\_project\_id | Project where service accounts and core APIs will be enabled. |
+| terraform\_sa\_email | Email for privileged service account. |
+| terraform\_sa\_name | Fully qualified name for privileged service account. |
 
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 
 ## Requirements
 
-These sections describe requirements for using this module.
-
 ### Software
 
-The following dependencies must be available:
+-   [gcloud sdk](https://cloud.google.com/sdk/install) >= 206.0.0
+-   [Terraform](https://www.terraform.io/downloads.html) >= 0.12.6
+-   [terraform-provider-google] plugin 2.1.x
+-   [terraform-provider-google-beta] plugin 2.1.x
 
-- [Terraform][terraform] v0.12
-- [Terraform Provider for GCP][terraform-provider-gcp] plugin v2.0
+### Permissions
 
-### Service Account
+- `roles/resourcemanager.organizationAdmin` on GCP Organization
+- `roles/billing.admin` on supplied billing account
+- Account running terraform should be a member of group provided in `group_org_admins` variable, to ensure that project creation and other API calls do not fail.
 
-A service account with the following roles must be used to provision
-the resources of this module:
+### Credentials
 
-- Storage Admin: `roles/storage.admin`
+If you are not customizing the credentials used in the variable `credentials_file_path`, please ensure you run `gcloud auth application-default login` prior to running your code to ensure the module can retreive credentials.
```

Comment:
This statement should be removed along with the associated variable.

Reply:
Agreed, will pull it out.